bbot/cli.py

```diff
@@ -91,6 +91,9 @@ def main():
                                     log.verbose(f'Enabling {m} because it has flag "{f}"')
                                     modules.add(m)
 
+                if not options.output_modules:
+                    options.output_modules = ["human"]
```

Comment:
How does this change interact with `bbot/scanner/scanner.py#60`? Which one takes precedence if no output module is specified?